doc/cookbook/type/create_custom_field_type.rst

```diff
@@ -0,0 +1,215 @@
+.. index::
+   single: Field; Custom field type
+
+How to Create a Custom Search Field Type
+========================================
+
+RollerworksSearch comes with a bunch of core field types available for
+building FieldSets. However there are situations where you may want to
+create a custom field type for a specific purpose. This recipe assumes
+you need a field definition which holds specially formatted Client ID's,
```

Comment:
thanks, i learned this http://englishplus.com/grammar/00000287.htm new thing today